runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargeConfig.java

```diff
@@ -34,273 +34,313 @@
 public interface ItemChargeConfig extends Config
 {
 	@ConfigItem(
-		keyName = "veryLowWarningColor",
-		name = "Very Low Warning Color",
-		description = "Configure the color of the overlay when charges are very low",
-		position = 1
+			keyName = "veryLowWarningColor",
```

Comment:
This PR is applying the wrong tab depth to every single `@ConfigItem` annotation in this file. Please fix the indentation. (only one level further than the `@ConfigItem`)

![tabs](https://user-images.githubusercontent.com/2199511/61250451-3aa63b00-a747-11e9-9c03-7a738a1abf1c.png)